test-gui/src/package/view-decoded-linear-position-plot/DecodedLinearPositionPlotView.tsx

```diff
@@ -101,24 +101,25 @@ const DecodedLinearPositionPlotView: FunctionComponent<DecodedLinearPositionProp
         context.imageSmoothingEnabled = false
         context.drawImage(canvas, displayRange[0], 0, displayRange[1] - displayRange[0], canvas.height, 0, 0, panelWidth, panelHeight)
         if (showObservedPositionsOverlay && scaledObservedPositions !== undefined) {
-            const verticalEpsilon = 0.005
+            const verticalEpsilonPx = 4
```

Comment:
Probably it would be better to analyze the data (and maybe the canvas dimensions) and see how big the dead regions are, and use that value (whether in percent, like we used to, or in pixels). But I didn't want to get into those weeds, so I just used this as a heuristic.

Reply:
Makes sense, thanks.